src/service/url-replacements-impl.js

```diff
@@ -639,6 +639,9 @@ export class UrlReplacements {
 
     /** @type {VariableSource} */
     this.variableSource_ = variableSource;
+
+    /** @type {Expander} */
```

Comment:
`{!Expander}`, since it will not be `null`

Reply:
changed type